sites/website/src/pages/index.jsx

```diff
@@ -106,6 +106,20 @@ HelloWorld.define({
                     <CodeBlock language="html">
                         {`<hello-world name="Earth"></hello-world>`}
                     </CodeBlock>
+
+                    <h3>Update your tsconfig.json file to support decorators</h3>
+                    <CodeBlock language="json">
+                      {{
+                        "compilerOptions": {
+                        "experimentalDecorators": true,
+                        "emitDecoratorMetadata": true,
+                        "module": "ESNext",
+                        "moduleResolution": "bundler",
+                        "target": "ES2020",
+                        "useDefineForClassFields": false
+                        }
+                      }}
+                    </CodeBlock>
```

Comment:
There's currently an error with the contents of the code block, it needs to be wrapped in back ticks instead of a double curly brace. Whitespace is also preserved in the rendered output so the formatting needs to be adjusted.

The only suggestion I have is to remove `emitDecoratorMetaData` since it is not required in order to use FAST, only if you are creating a library with it and want TypeScript to create the runtime type information for your decorators and even then it is optional.

The required tsconfig can technically be reduced even further to just `experimentalDecorators` and `useDefineForClassFields`, however the other options do provide a good starting point for most projects.

```suggestion
                    <CodeBlock language="json">
                      {`{
    "compilerOptions": {
        "experimentalDecorators": true,
        "module": "ESNext",
        "moduleResolution": "bundler",
        "target": "ES2020",
        "useDefineForClassFields": false
    }
}`}
                    </CodeBlock>
```